### PR TITLE
Chore: Split useUserInfoActions into small hooks

### PR DESCRIPTION
**Pull request by dougfabris**

### CODE REVIEW

**Review by gabriellsh — APPROVED** [review]

**Review by ggazzo — CHANGES_REQUESTED** [review]
image
missing actions for user card and contextualbar

File: apps/meteor/client/views/room/lib/getRoomDirectives.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { IRoom } from '@rocket.chat/core-typings';
 import { RoomMemberActions } from '../../../../definition/IRoomTypeConfig';
 import { roomCoordinator } from '../../../lib/rooms/roomCoordinator';
 
-export const getRoomDirectives = (room: IRoom): boolean[] => {
+export const getRoomDirectives = (room: IRoom): { [key: string]: boolean } => {
```
Comment (ggazzo) [inline apps/meteor/client/views/room/lib/getRoomDirectives.ts:6]: explicit put the keys here

**Review by ggazzo — CHANGES_REQUESTED** [review]

**Comment by d-gubert** [comment]
It doesn't seem that us from @RocketChat/apps can contribute in reviewing this PR 🤔

**Comment by dougfabris** [comment]
@d-gubert it was automatic for some reason, code owners 👀

**Comment by d-gubert** [comment]
> @d-gubert it was automatic for some reason, code owners 👀

I thought so, but I can't even see anything we're owners of 👀  Weird stuff

**Review by ggazzo — APPROVED** [review]

**Pull request merged**